### DISCUSSION TITLE
not able to load model using python code

### DISCUSSION THREAD

**Discussion by ayuraj736** [openai/whisper#2168] [Show and tell]
code I am using is 
import whisper

model = whisper.load_model("medium")
result = model.transcribe("/home/ayush_python/8ca36823-8c90-4abb-bbb0-45c076a34c47.mp3")
print(result["text"])

and when I am executing the python command 
 python3 whisper2.py
Traceback (most recent call last):
  File "/home/ayush_python/whisper2.py", line 1, in <module>
    import whisper
  File "/home/ayush_python/whisper.py", line 3, in <module>
    model = whisper.load_model("base")
AttributeError: partially initialized module 'whisper' has no attribute 'load_model' (most likely due to a circular import)

**Comment by glangford** [comment]
Looks like you have a file you created called "/home/ayush_python/whisper.py". Rename this file to something other than 'whisper.py' otherwise Python will be confused when you 'import whisper'.

**Reply by ayuraj736** [reply]
thanks for the help
